### PR TITLE
RPMIR-363

### PR DESCRIPTION
**Pull request by jhiggins-cpr**
First draft of Dingo 1.5 updates. Still waiting for a couple updates from @jberringer before being able to merge.

### CODE REVIEW

File: docs/robots/indoor_robots/dingo/maintenance_dingo.mdx
Comment (rhysfaultless-cpr) [inline docs/robots/indoor_robots/dingo/maintenance_dingo.mdx:1]: Units should be separated from the value: 14.6 V

**Review by rhysfaultless-cpr — APPROVED** [review]

File: docs/robots/indoor_robots/dingo/maintenance_dingo.mdx
Comment (jhiggins-cpr) [reply docs/robots/indoor_robots/dingo/maintenance_dingo.mdx:1]: Fixed

File: docs/robots/indoor_robots/dingo/user_manual_dingo.mdx [hunk 1]
```diff
@@ -425,26 +425,25 @@ The direction of the axes differs from those used for roll, pitch, and yaw in ai
...
+| Maximum Modules            | 2                                                                        | 4                                                              |
+| Maximum Payload            | 20 kg                                                                    | 25 kg                                                          |
+| Speed, Maximum             | 1.3 m/s                                                                  | 1.3 m/s                                                        |
+| Operating Environment      | Indoor                                                                   | Same as Dingo-D                                                |
+| Operating Time (Dingo 1.0) | 2 hours (1X SLA), 4 hours (1X Lithium)                                   | 2 hours (2X SLA), 4 hours (2X Lithium)                         |
+| Operating Time (Dingo 1.5) | 2 hours (1X SLA), 4 hours (1X LiFEPO<sub>4</sub>)                        | 2-6 hours (1-3X SLA), 3.5-11 hours (1-3X LiFEPO<sub>4</sub>)   |
```
Comment (rhysfaultless-cpr) [inline docs/robots/indoor_robots/dingo/user_manual_dingo.mdx]: Suggest:
2 - 6
1X - 3X

**Review by rhysfaultless-cpr — APPROVED** [review]

File: docs/robots/indoor_robots/dingo/user_manual_dingo.mdx (hunk 1 above)
Comment (jhiggins-cpr) [reply docs/robots/indoor_robots/dingo/user_manual_dingo.mdx]: Fixed

**Review by apopielas — APPROVED** [review]

**Pull request merged**